lib/plugins/aws/info/display.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     message += `${chalk.yellow('stack:')} ${info.stack}\n`;
     message += `${chalk.yellow('resources:')} ${info.resourceCount}`;
 
-    if (_.has(info, 'resourceCount') && info.resourceCount >= 150) {
+    if (info.resourceCount && info.resourceCount >= 150) {
```

Comment:
This can be put just as `if (info.resourceCount >= 150)` (if there's no `info.resourceCount` it'll naturally be _false_)